workshops/chrome-extension/background.js

```diff
@@ -0,0 +1,74 @@
+let timer;
+
+chrome.browserAction.setBadgeBackgroundColor({ color: '#777' });
+
+// retrieve and format timer text or play audio if time is up
+function getTimeLeft(endTime) {
+  const distance = new Date(endTime) - new Date();
+  if (distance <= 0) {
+    timerEnded();
+    return '';
+  }
+  const hours = Math.floor(distance / (1000 * 3600));
+  const minutes = Math.floor((distance % (1000 * 3600)) / (1000 * 60));
+  const seconds = Math.floor((distance % (1000 * 60)) / 1000);
+  let message = '';
+  // if multiple hours remain, format as "hh hrs" to prevent awkward UI
+  if (hours > 0) {
+    return `${hours} hrs`;
+  }
+  if (minutes >= 0) message += minutes === 0 ? `00:` : minutes < 10 ? `0${minutes}:` : `${minutes}:`;
+  if (seconds >= 0) message += seconds === 0 ? `00` : seconds < 10 ? `0${seconds}` : `${seconds}`;
+  return message;
```

Comment:
Instead of `let message` and doing string concatenation just return the result
``return `${Math.max(0, minutes).toString().padStart(2, "0")}:${Math.max(0, seconds).toString().padStart(2, "0")}`;``